source/core/src/main/com/csse3200/game/components/player/PlayerActions.java

```diff
@@ -43,8 +43,6 @@ public class PlayerActions extends Component {
   private SecureRandom random = new SecureRandom();
   int swordDamage = 5;
 
-  private static final String RIGHT_STRING = "right";
```

Comment:
I feel like this was probably done for code smells, maybe change all the directions to either be constants or enums?

Reply:
let me resolve my own conversations 😤